packages/ens-utils/src/price.ts

```diff
@@ -123,14 +128,44 @@ export const multiplyPriceByNumber = (price1: Price, price2: number): Price => {
   };
 };
 
+export const CurrencySymbology = {
+  /**
+   * The price will be displayed with the currency's acronym (e.g. USD).
+   */
+  Acronym: "Acronym",
+  /**
+   * The price will be displayed with the currency's symbol (e.g. $).
+   */
+  Symbol: "Symbol",
+} as const;
+export type CurrencySymbology =
+  (typeof CurrencySymbology)[keyof typeof CurrencySymbology];
+
+/**
+ * Returns a string containing the currency's symbol or acronym.
+ * @param currency: Currency. The currency to get the symbology for (e.g. Currency.Eth)
+ * @param symbology: CurrencySymbology. The symbology to use (e.g. CurrencySymbology.Symbol)
+ * @returns: string. The currency's symbol or acronym (e.g. "$" or "USD")
+ */
+export const getCurrencySymbology = (
```

Comment:
This idea doesn't belong inside `ens-utils`. It should move into `CurrencySymbol` in `namekit-react`.

Reply:
@lightwalker-eth maybe what we need is to rename it?

Because the same symbology rules are used in formattedPrice, an ens-utils feature